libs/ledgerjs/packages/hw-app-btc/src/splitTransaction.ts

```diff
@@ -17,13 +17,16 @@ export function splitTransaction(
   let nExpiryHeight = Buffer.alloc(0);
   let nVersionGroupId = Buffer.alloc(0);
   let extraData = Buffer.alloc(0);
+  let witnessScript, locktime;
   const isDecred = additionals.includes("decred");
   const isZencash = additionals.includes("zencash");
   const transaction = Buffer.from(transactionHex, "hex");
   const version = transaction.slice(offset, offset + 4);
   const overwinter =
     version.equals(Buffer.from([0x03, 0x00, 0x00, 0x80])) ||
-    version.equals(Buffer.from([0x04, 0x00, 0x00, 0x80]));
+    version.equals(Buffer.from([0x04, 0x00, 0x00, 0x80])) ||
+    version.equals(Buffer.from([0x05, 0x00, 0x00, 0x80]));
+  const zcashv5 = version.equals(Buffer.from([0x05, 0x00, 0x00, 0x80]));
```

Comment:
Just a suggestion but maybe we could use zcashv5 in the overwinter const so we don't need to repeat `Buffer.from([0x05, 0x00, 0x00, 0x80])`.

Could also be named isZcashV5 so we can see with just a glance it's a boolean.

Feel free to resolve if you don't find it relevant

Reply:
good point. I'll fix it